app/webfront/src/redux/user/slice.js

```diff
@@ -8,10 +9,19 @@ const userSlice = createSlice({
         profile: null,
         recipes: [],
         followers: [],
+        folowees: [],
         isLoading: false,
         error: null,
     },
-    extraReducers: () => {},
+    extraReducers: (builder) => {
+        builder
+            .addCase(getFollowers.fulfilled, (state, action) => {
+                state.followers = action.payload;
+            })
```

Comment:
If we receive followers as object we should adjust state and have extra field and set them properly
<img width="877" alt="Screenshot 2025-03-25 at 21 36 02" src="https://github.com/user-attachments/assets/5222ffd2-69e4-4d52-8b84-9721be4bb883" />